plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -8,3 +8,7 @@ deployStandardMenuLabel=Deploy to App Engine Standard...
 deployFlexCommandDescription=Uploads the project to Google App Engine flexible environment.
 deployFlexCommandName=Deploy to App Engine Flexible
 deployFlexMenuLabel=Deploy to App Engine Flexible...
+
+deployFlexExistingWarJarCommandDescription=Uploads pre-existing WAR or JAR file to Google App \
```

Comment:
the Google App...

Reply:
The message continues in the next line. It's a description, so it doesn't need to end with three dots either.

Reply:
Ah, you mean adding "the".